docs/en/operations/table_engines/replication.md

```diff
@@ -10,6 +10,11 @@ Replication is only supported for tables in the MergeTree family:
 - ReplicatedVersionedCollapsingMergeTree
 - ReplicatedGraphiteMergeTree
 
+ClickHouse uses [Apache ZooKeeper](https://zookeeper.apache.org) for storing replicas meta information.
```

Comment:
I think we should move this lower in this article, because it's detail that kind of ruins the high level introduction that follows